tooling/maven/camel-eip-documentation-enricher-maven-plugin/src/main/java/org/apache/camel/maven/DocumentationEnricher.java

```diff
@@ -106,7 +106,11 @@ private void addAttributeDocumentation(Log log, Element item, File jsonFile) thr
         if (!isNullOrEmpty(descriptionText)) {
             String text = descriptionText;
             if (!isNullOrEmpty(defaultValueText)) {
-                text += ". Default value: " + defaultValueText;
+                if (!text.endsWith(".")) {
```

Comment:
A liitle bit shorter syntax could be used instead
text += (text.endsWith(".") ? "" : ".") + " Default value: " + defaultValueText;